spec/unit/util_spec.rb

```diff
@@ -150,44 +150,7 @@ def withenv_utf8(&block)
     # Ruby retrieves / stores ENV names in the current codepage
     # when these tests no longer pass, Ruby has fixed its bugs and workarounds can be removed
 
-
-    # interestingly we would expect some of these tests to fail when codepage is 65001
-    # but instead the env values are in Encoding::ASCII_8BIT!
-    it "works around Ruby bug 8822 (which fails to preserve UTF-8 properly when accessing ENV) (Ruby <= 2.1) ",
```

Comment:
We're pulling 2.0 and 2.1 testing here?

Reply:
Yep, it's already been pulled. [Puppet will raise if ruby < 2.3](https://github.com/puppetlabs/puppet/blob/master/lib/puppet.rb#L4).